foodsaving/groups/models.py

```diff
@@ -120,6 +140,12 @@ def add_roles(self, roles):
         for role in roles:
             if role not in self.roles:
                 self.roles.append(role)
+                if role is GROUP_APPROVED_MEMBER:
+                    History.objects.create(
+                        typus=HistoryTypus.GROUP_JOIN,
+                        group=self.group,
+                        users=[self.user, ],
+                    )
```

Comment:
Hmm, not sure about this:
- we could consider joining the group and being approved to be distinct
- there are probably places that don't go through this add_role method (can it be done via signal instead?)
- I don't see the corresponding group leave history item for remove role

Reply:
Hmm, this might need more discussion then, e.g. what is the group join history type meant to do?
I see that I currently got it inconsistent.